### PR TITLE
Fix nxos_install_os test cases typo

### PR DESCRIPTION
**Pull request by pabelanger**
This should be ansible_connection, not connection_type.

Signed-off-by: Paul Belanger <pabelanger@redhat.com>

### CODE REVIEW

**Review by trishnaguha — CHANGES_REQUESTED** [review]
ansible_connection can only be local both for cli and nxapi transport.

File: test/integration/targets/nxos_install_os/tasks/main.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
 # Upgrade using SSH
 - include: network_cli.yaml
-  when: connection_type == 'network_cli'
+  when: ansible_connection == 'network_cli'
 - include: network_local.yaml
-  when: connection_type == 'cli_local'
+  when: ansible_connection == 'cli_local'
```
Comment (trishnaguha) [inline test/integration/targets/nxos_install_os/tasks/main.yaml]: Either we need to have one entrypoint for local connection tasks or drop cli local connection tests as we did for the other modules.

File: test/integration/targets/nxos_install_os/tasks/main.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 # Upgrade using NX-API
 - include: httpapi.yaml
-  when: connection_type == 'httpapi'
+  when: ansible_connection == 'httpapi'
 - include: nxapi.yaml
-  when: connection_type == 'nxapi_local'
+  when: ansible_connection == 'nxapi_local'
```
Comment (trishnaguha) [inline test/integration/targets/nxos_install_os/tasks/main.yaml]: To drop local cli test and fix test failure, this line needs to be changed to `when: ansible_connection == 'local'`

File: test/integration/targets/nxos_install_os/tasks/main.yaml (hunk 1 above)
Comment (trishnaguha) [reply test/integration/targets/nxos_install_os/tasks/main.yaml]: To drop local cli test, this block of code can be removed and the file tasks/network_local.yaml needs to be rm-ed.
```
- include: network_local.yaml
  when: ansible_connection == 'cli_local'
```

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now 27893c349d4af7f93063ed5b6346bca9946aad7a

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now acadca385d04ab8d92bdbe31a4ce9bf47ea5b334

**Review by trishnaguha — APPROVED** [review]

**Pull request merged**